org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -817,6 +817,21 @@ class ServerCapabilities {
 	 * The server provides rename support.
 	 */
 	Boolean renameProvider
+	
+	/**
+	 * The server provides document link support.
+	 */	
+	DocumentLinkOptions documentLinkProvider
+	
+	/**
+	 * The server provides execute command support.
+	 */
+	ExecuteCommandOptions executeCommandProvider
+	
+	/**
+	 * Experimental server capabilities.
+	 */
+	Object experimental
```

Comment:
I see the `experimental` field only for the ClientCapabilities?
https://github.com/Microsoft/language-server-protocol/blob/dbaeumer/3.0-spec/protocol.md